tests/conditional_compilation/test_size_tracking.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+import logging
+
+from path_utils import library_name  # pylint: disable=import-error
+
+
+def get_lib_sizes(path, libraries):
+    """Function for getting lib sizes by lib names"""
+    assert Path.exists(path), f'Directory {path} isn\'t created'
+    result = {}
+    error_lib = []
+    for lib in libraries:
+        try:
+            result[lib] = Path(path).joinpath(library_name(lib)).stat().st_size
+        except FileNotFoundError:
+            error_lib.append(lib)
+    assert len(error_lib) == 0, 'Following libraries couldn\'t be found: {}'.format(error_lib)
```

Comment:
it also would be great to log paths which are not found as files. Now only libs names will be presented in failure message

Reply:
Currently output show follow message:
![image](https://user-images.githubusercontent.com/76170835/111126108-5ea4ca00-8583-11eb-85e1-fd64580e0b7d.png)